### PR TITLE
feature/OP-818: The History Section

### PR DESCRIPTION
**Pull request by Palisand**

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -45,7 +46,7 @@ class Roles(db.Model):
     __tablename__ = 'roles'
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String(64), unique=True)
-    permissions = db.Column(db.Integer)
+    permissions = db.Column(db.Integer)  # TODO: bigint + migrate
```
Comment (gzhou1) [inline app/models.py]: Change to bigint?

File: app/request/api/views.py [hunk 2]
```diff
@@ -51,12 +52,12 @@ def edit_privacy():
     if title is not None:
         privacy['title'] = title == 'true'
         previous_value['privacy'] = current_request.privacy['title']
-        new_value['privacy'] = title
+        new_value['privacy'] = privacy['title']
```
Comment (Palisand) [inline app/request/api/views.py]: No, this is because `title` is a string, whereas `privacy['title']` is a bool.

File: app/models.py (hunk 1 above)
Comment (Palisand) [inline app/models.py]: Yep, might as well do it here than in the branch I'm currently working on.

File: app/request/api/views.py (hunk 2 above)
Comment (gzhou1) [inline app/request/api/views.py]: Ah right. Good catch.

**Pull request merged**